src/rgw/rgw_rest_pubsub.cc

```diff
@@ -381,15 +591,15 @@ RGWOp* RGWHandler_REST_PSTopic_AWS::op_post() {
   if (s->info.args.exists("Action")) {
```

Comment:
might want to consider (maybe not in this PR) to return a u.p

Reply:
could make sense. however, this is an API used all over the RGW.
cannot be modified in this context